code/datums/components/mirage_border.dm

```diff
@@ -4,7 +4,7 @@
 /datum/component/mirage_border/Initialize(turf/target, direction, range=world.view)
 	if(!isturf(parent))
 		return COMPONENT_INCOMPATIBLE
-	if(!target || !istype(target) || !direction)
```

Comment:
undocumented change?

Reply:
`Also contains a typo fix and removes an impossible if test from mirage_border`

Reply:
what was i thinking yesterday

Reply:
oh whatever the code actually works but i did not think this through